types/marked/marked-tests.ts

```diff
@@ -247,6 +245,18 @@ marked.use({
     extensions: [tokenizerExtension, rendererExtension, tokenizerAndRendererExtension],
 });
 
+const asyncExtension: marked.MarkedExtension = {
+    async: true,
+    async walkTokens(token) {
+        if (token.type === 'code') {
+            await Promise.resolve(3);
+            token.text += 'foobar';
+        }
+    },
+};
```

Comment:
This is the actually relevant test. Other diffs in this file are from running `prettier`